events/event_handler.go

```diff
@@ -160,6 +161,18 @@ func (fh *eventHandler) handleStatusEvent(body []byte) error {
 		"TaskStatus": task.TaskStatus,
 	}).Info("Got StatusEvent")
 
+	app, err := fh.marathon.App(task.AppID)
```

Comment:
Right now, when a "Killing" or "Failed" event happen, it try to deregister application even if it's not consul.

We have a lot of `level=error msg="There was a problem deregistering task" Id={APP}.e41a91e6-3be7-11e7-bcb0-c4346bb6b3b0 error="Couldn't find any service matching task id {APP}.e41a91e6-3be7-11e7-bcb0-c4346bb6b3b0"`